source/PlatformerHero.hx

```diff
@@ -0,0 +1,63 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+
+/**
+ * ...
+ * @author Cory Jackson
+ */
+class PlatformerHero extends FlxSprite 
+{
+	var colorArray = [FlxColor.RED, FlxColor.YELLOW];
+	var i = 0;
+	
+	public function new(?X:Float=0, ?Y:Float=0, ?SimpleGraphic:FlxGraphicAsset) 
+	{
+		super(X, Y, SimpleGraphic);
+		this.makeGraphic(25, 150, FlxColor.YELLOW);
+		this.acceleration.y = 60;
+		this.maxVelocity.y = 150;
+	}
+	
+	override public function update(elapsed:Float):Void
+	{
+		var leftPressed:Bool = FlxG.keys.pressed.LEFT;
+		var rightPressed:Bool = FlxG.keys.pressed.RIGHT;
+		var upPressed:Bool = FlxG.keys.pressed.Z;
```

Comment:
I think this variable name is more than a little misleading. Something to consider changing.